redux-localstorage/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for redux-localstorage 1.0
+// Project: https://github.com/elgerlambert/redux-localstorage
+// Definitions by: Karol Janyst <https://github.com/LKay>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as Redux from "redux";
+
+export interface ActionTypes {
+    INIT: string;
+}
+
+export type AdapterCallback = <A>(err?: any, result?: A) => void;
+
+export interface StorageAdapter<A> {
+    0: A;
+    put(key: string, value: any, callback: AdapterCallback): void;
+    get(key: string, callback: AdapterCallback): void;
+    del(key: string, callback: AdapterCallback): void;
+}
+
+export type StorageAdapterCreator<A> = (storage: A) => StorageAdapter<A>;
+
+export interface StorageAdapterEnhancer {}
+
+export function mergePersistedState (merge?: <A1, A2>(initialState: A1, persistentState: A2) => A1 & A2): <A>(next: Redux.Reducer<A>) => Redux.Reducer<A>;
```

Comment:
For Redux ^4.0.0, there's a additional `AnyAction` for `Reducer`. The latter part should be `<A,AnyAction>(next: Redux.Reducer<A,AnyAction>) => Redux.Reducer<A,AnyAction>;`